httpserver/src/test/kotlin/tbdex/sdk/httpserver/handlers/GetExchangesTest.kt

```diff
@@ -0,0 +1,95 @@
+package tbdex.sdk.httpserver.handlers
+
+import ServerTest
+import TestData
+import io.ktor.client.plugins.auth.*
+import io.ktor.client.plugins.auth.providers.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
```

Comment:
FYI more implicit imports on this line and the lines bove, not sure if this is intentional -- I recall one such rule being something like, "_if > N number of imports from the given namespace then wildcard it_"

Commenting on b/c of the commentary [above](https://github.com/TBD54566975/tbdex-kt/pull/123/files#r1433587912)

Reply:
@jiyoontbd I think my IJ messed up your imports when I made changes to the test?

Reply:
@KendallWeihe It's usually a stylistic choice and we just want to be consistent across the codebase. IIRC there are some compilation time gains by using explicit imports vs wildcard, but I haven't personally dug too much into that!

Reply:
I for one prefer the explicit imports, but yeah beauty is in the eye of the beholder